src/faces/FacesTool.vala

```diff
@@ -337,8 +337,10 @@ public class FacesTool : EditingTools.EditingTool {
                 Process.spawn_sync(null, argv, null, SpawnFlags.STDERR_TO_DEV_NULL, null, out output);
 
             } catch (SpawnError e) {
-                critical("Error trying to spawn face detection program: %s\n", e.message);
-                assert_not_reached();
+                output = "Error trying to spawn face detection program: " + e.message + "\n";
```

Comment:
If this is shown to the user, it needs to be marked translatable. Why don't you save the SpawnError directly? Then you only have to do a != null check below.